activerecord/test/cases/relation/where_chain_test.rb

```diff
@@ -2,17 +2,31 @@
 
 require "cases/helper"
 require "models/post"
+require "models/author"
 require "models/comment"
+require "models/categorization"
 
 module ActiveRecord
   class WhereChainTest < ActiveRecord::TestCase
-    fixtures :posts
+    fixtures :posts, :authors
 
     def setup
       super
       @name = "title"
     end
 
+    def test_missing_with_association
```

Comment:
What about to introduce more high-level test, not one testing the implementation?


```ruby
post_without_author = Post.create(author: nil)
assert_equal [post_without_author], Post.without(:author).to_a
```